### ISSUE TITLE
Error on updating MBR

### ISSUE THREAD

**Issue by AssassinAI** [corpnewt/gibMacOS#71]
Script works fine until it reaches this stage, where it spits out the attached error. Closing the error box continues to the next stage with PBR but spits out the same error. Any ideas? 
![Error](https://user-images.githubusercontent.com/17803607/72935280-f1b9e980-3d5c-11ea-8e2d-685da557d86e.png)

**Comment by DeSchmiddi** [comment]
Same here

**Comment by cyanit** [comment]
![image](https://user-images.githubusercontent.com/26248671/80278556-b6caa780-86f7-11ea-91e2-ceb8aad9f5f8.png)

same error, but this pops up and clicking on restore does nothing. Increasing the sectors is not possible. Clicking cancel closes the dialog.

Well lets see if this works anyway....

**Comment by thvozdovic** [comment]
Same here

**Comment by adamnickster** [comment]
Same here. First run created USB without problem and next day pops that error. Tried on two PCs Windows 10 both.

**Comment by zk-g** [comment]
Same anyone found a solution?

**Comment by thvozdovic** [comment]
Nope, and it still doesnt work even on the newest freshly installed Win10

**Comment by thvozdovic** [comment]
![](https://i.imgur.com/215pnqi.png)

**Comment by ftln94220** [comment]
Hopefully get this fixed, need to reinstall my Catalina :(

**Comment by adamnickster** [comment]
If you want temporary fix replace link in MakeInstall.py on line 35 to: `self.oc_boot_url = "https://github.com/acidanthera/OpenCorePkg/raw/a1f60694c7c5fb07392529fa600d939dd916a81b/Utilities/BootInstall/"` commit before 24th April

**Comment by ftln94220** [comment]
Thanks :) Trying now

**Comment by corpnewt** [comment]
@adamnickster I pushed that as a temp change while waiting to see if Acidanthera's Duet repo will supply releases.

-CorpNewt

**Comment by makewheels** [comment]
same issue, dont know what to do

**Comment by makewheels** [comment]
> If you want temporary fix replace link in MakeInstall.py on line 35 to: `self.oc_boot_url = "https://github.com/acidanthera/OpenCorePkg/raw/a1f60694c7c5fb07392529fa600d939dd916a81b/Utilities/BootInstall/"` commit before 24th April

doesnt work

**Comment by makewheels** [comment]
I found a way,the project maybe have some bugs and cant run in python,we can run in bat file
download two files in opencore 
boot0 and boot1f32
put BOOTICEx64.exe in the same folder

run the script manually：
attention: modify the device number
BOOTICEx64.exe /device=3 /mbr /restore /file=boot0 /keep_dpt
BOOTICEx64.exe /device=3:0 /pbr /restore /file=boot1f32 /keep_bpb

**Comment by lgibson02** [comment]
This error can also appear on 32-bit systems because BOOTICEx64 will not run on those (64-bit binary). It took me a good twenty minutes to figure that out by debugging though. The way errors are picked up upon in this project is quite poor and they often end up propagating instead of being handled.
